sfcli/SmartFace.Cli/Core/Domain/WatchlistMember/Impl/RegisterWatchlistMemberExtendedJsonLoader.cs

```diff
@@ -23,7 +23,11 @@ public WatchlistMemberMetadata[] GetWatchlistMemberRegistrationData(string path)
                 throw new ProcessingException($"Directory does not exists {path}");
             }
 
-            var files = Directory.GetFiles(path, "*.json");
+            var files = Directory.GetFiles(path, "*.json", new EnumerationOptions
+            {
+                MatchCasing = MatchCasing.CaseInsensitive
```

Comment:
why this was added?

Reply:
According to a test this code is supposed to handle both .json and .JSON (and any other) casing of the extension. Windows filesystem is not case sensitive, so it worked there, but on Linux we needed to explicitly specify the case-insensitive behavior